### ISSUE TITLE
Documentation should be changed to avoid "pattern.en" as a module name

### ISSUE THREAD

**Issue by glarchev** [clips/pattern#136]
It looks like some of the examples in the documentation use "pattern.en" as a module name -- it appears that it should actually be "pattern.text.en" instead.

**Comment by verhoevenben** [comment]
No, all language modules were designed by the developers to work as a module on their own (so pattern.en or pattern.it, …) for ease of use, I suppose.
For efficiency of the code, they gathered all language modules under the module ‘text’.

**Comment by glarchev** [comment]
Since all the language modules are nested under "text", I think it's more confusing to refer to them as just "pattern.en". Although it appears to work, my IDE (PyDev) complains about unresolved import.
